Dockerfile

```
@@ -5,4 +5,5 @@ WORKDIR /code
 COPY requirements.txt .
 RUN pip3 install -r requirements.txt --no-cache-dir
 COPY . .
+ENV PYTHONPATH "${PYTHONPATH}:/code/src"
 CMD uvicorn src.run_webhook_api:api --host 0.0.0.0 --port 8000 --reload
```

попробуй так сделать
```suggestion
CMD uvicorn run_webhook_api:api --host 0.0.0.0 --port 8000 --reload
```